torch/csrc/generic/StorageMethods.cpp

```diff
@@ -190,13 +190,33 @@ PyObject * THPStorage_(newWithFile)(PyObject *_unused, PyObject *file)
   int fd = PyObject_AsFileDescriptor(file);
   THPUtils_assert(fd != -1, "_new_with_file couldn't retrieve a file "
       "descriptor from given object");
-  THStoragePtr storage = THPStorage_(readFileRaw)(fd);
+  THStoragePtr storage = THPStorage_(readFileRaw)(fd, nullptr);
   PyObject *result = THPStorage_(New)(storage);
   storage.release();
```

Comment:
I think that this smart pointer is no longer necessary (`THPStorage_(New)` now steals the reference even if it fails).